config/jjb-templates/project-func-matrix.yaml

```diff
@@ -147,3 +147,41 @@
             TEST_PATH=$TEST_PATH
             GIT_BRANCH=$GIT_BRANCH
             DISPLAY_NAME=$GIT_REPO/$GIT_BRANCH $TEST_PATH
+
+- job:
+    name: test_func_openstack_mysql8_ovn_matrix
+    disabled: true
+    project-type: matrix
+    description: |
+        <p><i>Dynamically Generated Job - Do not edit through the Jenkins Web UI.  You will lose your changes.</i></p>
+    execution-strategy:
+      sequential: true
+    node: task
+    triggers:
+        - timed: ""
```

Comment:
To actually enable scheduled testing, this will need something like https://github.com/openstack-charmers/bot-control/blob/master/config/jjb-templates/project-mojo-matrix.yaml#L360